### ISSUE TITLE
Winit version 0.20.0 for Vulkano-win

### ISSUE THREAD

**Issue by the10thWiz** [vulkano-rs/vulkano#1295]
The `vulkano-win` crate doesn't support the latest winit version, 0.20.0.

Updating the `vulkano-win` crate to the latest version should only require updating the use statements, but it will cause breaking changes.

At the very least, I would like to know why it hasn't been updated yet. I am also willing to update and test it myself. Thanks.

**Comment by jmi2k** [comment]
What a coincidence, I faced the same problem yesterday! It seems it was planned for this version, but it was reverted by 2e6868a because of the issues the nannou project was having (linked in #1267).

Right now I'm just using [TethysSvensson's fork](https://github.com/TethysSvensson/vulkano) which just does what you say while I wait for the next release which (hopefully, given their intention to upgrade) will use the latest version of winit.

**Comment by AustinJ235** [comment]
I can possibly work in the patch tonight after work getting a bit late in the morning. I will probably hold on release until I confirm the fix with pipeline barriers.

As for why I am late on this, I thought they were still in the alpha stages of the release.

**Comment by dbartussek** [comment]
An issue I have noticed with the current version is that `VkSurfaceBuild::build_vk_surface` takes `&EventLoop<()>` as its argument. This makes it not possible to create a new window in `EventLoop::run`.
An easy fix is to replace `EventLoop` with `EventLoopWindowTarget<T>` where T can be anything. (I implemented this for my use case in https://github.com/dbartussek/vulkano/commit/54fc55f6a5d86f706ad08d3075f96e6fb5fcf4b5)
